Description: Use python3-sip module, not the bundled copy of PyQt5
 Starting with 5.11, PyQt5 bundles a convenience copy of the
 sip library. The Debian package python3-pyqt5 does not use this
 copy, therefore the import statement below fails. We revert
 back to the separately distributed python3-sip package, using
 the statement "import sip".
Forwarded: not-needed
Last-update: 2018-10-20
Author: Antoine Beaupre <anarcat@debian.org>

--- a/raphodo/rapid.py
+++ b/raphodo/rapid.py
@@ -89,10 +89,7 @@
 )
 from PyQt5.QtNetwork import QLocalSocket, QLocalServer

-if LooseVersion(QtCore.PYQT_VERSION_STR) >= LooseVersion('5.11'):
-    from PyQt5 import sip
-else:
-    import sip
+import sip

 from raphodo.storage import (
     ValidMounts, CameraHotplug, UDisks2Monitor, GVolumeMonitor, have_gio,
